src/components/ButtonWithDropdownMenu.js

```diff
@@ -55,25 +55,23 @@ const ButtonWithDropdownMenu = (props) => {
     const {windowWidth, windowHeight} = useWindowDimensions();
     const caretButton = useRef(null);
 
-    const measurePopoverPositionAndShowMenu = () => {
+    const measurePopoverPositionAndShowMenu = (showMenu) => {
```

Comment:
make the default `showMenu` value `true` to be consist with the function name.

Reply:
@s77rt  done